docker/test/integration/minifi/test/__init__.py

```diff
@@ -166,20 +184,33 @@ def log_nifi_output(self):
             stats = container.stats(stream=False)
             logging.info('Container stats:\n%s', stats)
 
-    def check_output(self, timeout=5, **kwargs):
```

Comment:
I think having kwargs here is useful, as kwargs can contain subdir as a key.

Reply:
I removed it as `subdir` was the only kwarg we used and replaced it with the `subdir` parameter with empty default value, which is a more explicit signature to see what args we can use.  
`    def check_output(self, timeout=5, subdir=''):`

We can still use it as a kwarg when calling the check_output.
`        assert cluster.check_output(30, subdir='success')`

Reply:
Okay, works for me.